packages/file_selector/file_selector/example/.gitignore

```diff
@@ -42,7 +42,6 @@ app.*.map.json
 
 # Currently only web supported
 android/
```

Comment:
I don't know why these lines were even here; we don't normally ignore directories that don't exist. Let's remove android/ too and save some confusion for whoever adds Android support.